tests/api/test_api_osp_deployment.py

```diff
@@ -0,0 +1,244 @@
+import re
+import pytest
+import string
+import random
+
+from lib.api import fusor_api
+
+
+def parse_ip_from_url(url):
+    """
+    Use a regular expression to parse the IP from a url.
+    Will return IP address if not protocol is specified
+    This assumes that the url does not use a DNS address
+    """
+    # Regex groups 1 and 2 will make up the protocol and separator
+    # Group 3 will be the IP address
+    # Group for will be everything after the IP
+    url_pattern = r'((https?)://)?(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})(.*)'
+    re_sult = re.match(url_pattern, url)
+
+    protocol, proto_full, ip, url_tail = re_sult.groups()
+
+    return ip
+
+
+@pytest.fixture(scope="module")
+def fusor_admin_username(variables):
+    return variables['credentials']['fusor']['username']
```

Comment:
Nice! If these wind up being needed for tests in other files, they could be moved to the conftest/ directory in the future.